### DISCUSSION TITLE
Инструкция по запуску

### DISCUSSION THREAD

**Discussion by Mixxxar** [WLM1ke/poptimizer#52] [General]
Сам не спец, прошу дайте инструкцию по клонированию и дальнейшему тестированию возможностей этого проекта.

**Comment by WLM1ke** [comment]
Этот проект не является готовым продуктом адаптированным под неподготовленного человека. Могу дать только инструкции общего порядка.
Для работы нужно скопировать файлы проекта удобным для вас способом, установить MongoDB и MongoDB Database Tools, Python 3.9 и все зависимости из requirements.txt. Запуск реализован через CLI:
python3 -m  poptimizer
Там можно посмотреть перечень команд и help к ним. А дальше только самому копаться и разбираться в коде.

**Reply by Mixxxar** [reply]
Спасибо! Дальше буду стараться самостоятельно разобраться!

**Reply by WLM1ke** [reply]
Удачи. Если совсем с разработкой не знакомы, то будет очень непросто. 
Лучше начать с основ - изучения Python, настройки виртуального окружения и основ работы с git.

**Reply by Mixxxar** [reply]
Таким путем и иду) Спасибо за совет и искренних успехов!

**Reply by robotbull** [reply]
> Таким путем и иду) Спасибо за совет и искренних успехов!

Приветствую! Тоже начал разбираться. Удалось разобраться?
